package.json

```diff
@@ -14,23 +13,34 @@
     "!dist/**/*.{stories,test}.*",
     "!dist/setupTests.*"
   ],
-  "engines": {
-    "node": "~20.9.0"
```

Comment:
Question: is it not still helpful to specify the node version here? Thought this was needed to get compatibility warnings when adding new packages?

Reply:
Ah yeah, I had removed it while just futzing around getting things to work; will add back with the bumped version

Reply:
Do we require all end-users to be on Node 22 (exactly)? It feels like the `engines` property we're specifying is too restrictive for a public package. Typically, you'll see something like this:

```json
"engines": {
  "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
}
```